scripts/pi-hole/php/loginpage.php

```diff
@@ -30,8 +30,8 @@
         <div class="row">
           <div class="col-xs-8 hidden-xs hidden-sm">
           <ul>
-            <li><samp>Return</samp> &rarr; Log in and go to requested page (<?php echo $scriptname; ?>)</li>
-            <li><samp>Ctrl+Return</samp> &rarr; Log in and go to Settings page</li>
+            <li><code>Return</code> &rarr; Log in and go to requested page (<?php echo $scriptname; ?>)</li>
```

Comment:
These could be `<kbd>`. Let me know if you want to switch to `kbd`

Reply:
I agree. Though for the second one, put it around each key like so, rather than the whole chord?

![image](https://user-images.githubusercontent.com/1998970/70463628-262b5c00-1ab5-11ea-9a92-bb61c5aa3eb6.png)

Reply:
Yup that's what I had in mind